product_docs/docs/epas/16/upgrading/minor_upgrade/index.mdx

```diff
@@ -1,18 +1,18 @@
 ---
-title: "Minor version upgrade with package manager"
-description: "Describes using a package manager to perfom a minor version upgrade of EDB Postgres Advanced Server."
+title: "Minor version upgrade with a package manager"
+description: "Describes using a package manager to perform a minor version upgrade of EDB Postgres Advanced Server."
 navigation:
 - 05_performing_a_minor_version_update_of_an_rpm_installation
-- performing_a_minor_version_update_of_Deb_installation
+- performing_a_minor_version_update_of_deb_installation
```

Comment:
I am not sure if the URLs are case-sensitive. The file name contains Deb (with uppercase D), so I think it would be safer to maintain the same name here. 
Alternatively, we could change this reference AND the file name.

Reply:
I can confirm that changing this becomes an issue for linking as warned by the link checker: 

<img width="1399" alt="Screenshot 2024-08-05 at 13 07 55" src="https://github.com/user-attachments/assets/e2071b0d-7a1e-4081-9d9a-5bf9184e872c">

Which means we have to roll these back, or change the name of the original file with lowercase for consistency. I am fine with both :)